app/pb_kits/playbook/pb_person_contact/_person_contact.html.erb

```diff
@@ -1,25 +1,26 @@
 <%= content_tag(:div,
-    id: object.id,
-    data: object.data,
-    class: object.classname) do %>
-    <%= pb_rails("person", props: {
-        first_name: object.first_name,
-        last_name: object.last_name,
+  id: object.id,
+  data: object.data,
+  class: object.classname) do %>
+  <%= pb_rails("person", props: {
+      first_name: object.first_name,
+      last_name: object.last_name,
+  }) %>
+  <% object.valid_contacts.each do |contact| %>
+    <%= pb_rails("contact", props: {
+        contact_type: contact[:contact_type],
+        contact_value: contact[:contact_value],
+        contact_detail: contact[:contact_detail],
     }) %>
-    <% object.contacts.each do |contact| %>
+  <% end %>
+  <% unless object.wrong_contacts.empty? %>
+    <div class="wrong_numbers"> <%= pb_rails("caption", props: { text: "wrong number" }) %></div>
```

Comment:
Why wrap this in a div, and not add a class to the wrong number caption?

If we want Playbook to be generally useful to others, what are our thoughts on handling copy like "wrong number"? This might not be the way others want to describe this, and others might not be able to only use English. I would consider moving things like "wrong number" out of hard coded HTML and make it configurable. I think taking advantage of translations would a good approach. This line could be turned into:

```suggestion
    <div class="wrong_numbers"> <%= pb_rails("caption", props: { text: t("pb_kits/playbook/pb_person_contact/wrong_number") }) %></div>
```

Now others can define alternative language versions of this, and can redefine what should display for English.

Also, do we have a convention for custom css rules? I usually see classes written as "wrong-numbers".

Reply:
@terryfinn I'm new to the repo but I think if we want to go in this path, we need to establish it as a standard first and then start doing it.

Maybe adding a `CONTRIBUTING.md` file with such standards would be helpful so that everyone knows what guidelines they should be following.

Maybe a template for PRs with a checklist would also be helpful.

Thoughts?

Reply:
Totally forgot that we can pass that `classname` prop and get that style on there. Got that and the conditional issue below resolved in the latest commit. I like `if` better than `unless` as well.

Reply:
@terryfinn - As flexible and inclusive as the translations would be, I feel like that would be out of scope here for this PR. Solidifying a standard going forward, as well as a template like @rafbgarcia pointed out would be awesome too.